packages/python/test/integration-setup.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+const { intoChunks } = require('../../../utils/chunk-tests');
+
+const {
+  testDeployment,
+} = require('../../../test/lib/deployment/test-deployment.js');
+
+jest.setTimeout(4 * 60 * 1000);
+
+module.exports = function setupTests(groupIndex) {
+  const fixturesPath = path.resolve(__dirname, 'fixtures');
+  const testsThatFailToBuild = new Map([
+    ['30-fail-build-invalid-pipfile', 'Unable to parse Pipfile.lock'],
+    [
+      '31-fail-build-invalid-python36',
+      'Python version "3.6" detected in Pipfile.lock is discontinued and must be upgraded.',
+    ],
+  ]);
+  const allFixtures = fs.readdirSync(fixturesPath);
+
+  let chunkedFixtures = allFixtures;
+  if (typeof groupIndex !== 'undefined') {
+    chunkedFixtures = intoChunks(1, 2, allFixtures)[groupIndex - 1];
+
+    console.log('testing group', groupIndex, chunkedFixtures);
+  }
+
+  // filter out '00-request-path' because it has special handling in "integration-1.test.ts"
+  // we wait until after chunking to filter it out so that chunk 1 will have one less test in the chunk
+  // which is backfilled by the special handling
```

Comment:
I actually don't think it needs to be filtered out. I think one is testing dev and one is testing deploy

Reply:
Good point. Updated: [`2057ab7` (#9771)](https://github.com/vercel/vercel/pull/9771/commits/2057ab7626539ba36faa369ea29897eaab8ec942) + [`caf648d` (#9771)](https://github.com/vercel/vercel/pull/9771/commits/caf648d71db1a1ffde1e491420d8b436adf57d6b)